code/modules/admin/verbs/possess.dm

```diff
@@ -25,18 +25,24 @@
 	usr.control_object = O
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Possess Object") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
-/proc/release()
+/proc/release(obj/O in world) //yogs
 	set name = "Release Obj"
 	set category = "Object"
 	//usr.loc = get_turf(usr)
 
-	if(usr.control_object && usr.name_archive) //if you have a name archived and if you are actually relassing an object
-		usr.real_name = usr.name_archive
-		usr.name_archive = ""
-		usr.name = usr.real_name
-		if(ishuman(usr))
-			var/mob/living/carbon/human/H = usr
-			H.name = H.get_visible_name()
+//YOGS start
+	if(!usr.control_object)
+		to_chat(usr, "You need to possess an object first!")
+		return
+//YOGS end
+
+		if(usr.control_object && usr.name_archive) //if you have a name archived and if you are actually relassing an object

```

Comment:
This line is indented 2 twice, while the `if(!usr.control_object)` is only indented once, meaning this only triggers if they don't have have a control_object (even though there's a return in front of it)